app/views/profiles/edit.html.erb

```diff
@@ -14,6 +14,9 @@
   <%= f.label :desires %>
   <%= f.text_area :desires %>
 
+  <%= f.check_box :smoker %>  
```

Comment:
This is fine but atm there isn't any semantic markup at all in the form. I suggest wrapping each of these fields in a fieldset, or a ul/ li list